src/main/resources/explanations/challenge1.adoc

```diff
@@ -2,6 +2,8 @@
 
 When people write a Proof of Concept, they often start with hardcoded secrets, such as a `password` in code. What if we forget to remove these hardcoded secrets?
 
-Can you spot the secret we are looking for in code? What about looking for it at the container?
+Can you spot the secret we are looking for in code? What about looking for it in the container?
 
-You can spot the secret easily by using https://github.com/trufflesecurity/truffleHog[*Trufflehog*] and https://github.com/awslabs/git-secrets[*Git-secrets*]. Just dive into the code!
+Sometimes the simpler tools are the most effective. Try cloning the repo and use https://man7.org/linux/man-pages/man1/grep.1.html[*grep*] to see what you find. 
+
+You can also find the answer using https://github.com/awslabs/git-secrets[*Git-secrets*] once you have added this rule `git secrets --add 'password\s*=\s*.+'`. Just dive into the code!
```

Comment:
Can you please update challenge1_hint.adoc in which git-secrets is setup as a third step? that way everyone can use it :)

Reply:
I think this is done. Let me know what you think of the new layout